public/app/features/dashboard-scene/settings/variables/components/QueryVariableForm.tsx

```diff
@@ -67,6 +67,13 @@ export function QueryVariableEditorForm({
   const { value: dsConfig } = useAsync(async () => {
     const datasource = await getDataSourceSrv().get(datasourceRef ?? '');
     const VariableQueryEditor = await getVariableQueryEditor(datasource);
+    const defaultQuery = datasource?.getDefaultQuery?.(CoreApp.Dashboard);
```

Comment:
@ivanortegaalba based on @kevinwcyu comment I found what was wrong:

```suggestion
    const defaultQuery = datasource?.variables?.getDefaultQuery?.();
```